crates/virtio-queue/src/queue.rs

```diff
@@ -266,13 +267,24 @@ impl<M: GuestAddressSpace, S: QueueStateT> Queue<M, S> {
     pub fn set_next_used(&mut self, next_used: u16) {
         self.state.set_next_used(next_used);
     }
+
+    /// Pop and return the next available descriptor chain, or `None` when there are no more
+    /// descriptor chains available.
+    pub fn pop_descriptor_chain(&mut self) -> Option<DescriptorChain<M::T>> {
```

Comment:
Hmm the commit message says there are no methods added to `Queue`, but looks like there are some. Am I missing something?

Reply:
You are definitely right, some methods from a previous iteration actually slipped through, I'll remove when with the next push, thanks for catching this!